aiohttp/http_websocket.py

```diff
@@ -667,11 +667,15 @@ async def pong(self, message: bytes = b"") -> None:
             message = message.encode("utf-8")
         await self._send_frame(message, WSMsgType.PONG)
 
-    async def ping(self, message: bytes = b"") -> None:
+    async def ping(self, message: bytes = b"", consume_errors: bool = False) -> None:
```

Comment:
I also think that it is possible to change the type of the message parameter to Union[bytes, str] in order to increase the flexibility of using the function and reduce the need for type conversion.

Reply:
I don't think that has anything to do with the PR though..

Reply:
https://github.com/aio-libs/aiohttp/pull/7266